### ISSUE TITLE
Format ROW Emails (SS) (Bugs on PROD)

### ISSUE THREAD

**Issue by vschandramouli** [cityofaustin/atd-data-tech#11662]
Splitting card #11025 and creating this one to standardize Subject lines across all ROW emails as requested by Paloma. 

**Acceptance Criteria**

The AC provided in this card applies for DS, EX, RW and PSM folders. 

**For the emails Sent from Department Comments Attempt on Web Application Acceptance process, the subject line of the email should be set to the following format:**
-- ROW COMMENTS – PERMIT NUMBER + TYPE – FOLDER NAME
-- For example, ROW COMMENTS – 2019-212899 EX – ATD Signals Corpus Christi Dr 6700Blk modifications 

**For the emails sent from Accepted or Rejected attempts on the Web Application Acceptance process, the subject line of the email should be set to the following format:**
-- ROW APPLICATION STATUS UPDATE – PERMIT NUMBER + TYPE – FOLDER NAME
-- For example, ROW PERMIT STATUS UPDATE – 2019-212899 EX – ATD Signals Corpus Christi Dr 6700Blk modifications 

**For the Invoice emails sent from the Fee tab -> Print Bill option, the subject line of the email should be set to the following format:**
-- ROW INVOICE – PERMIT NUMBER + TYPE – FOLDER NAME
-- For example, ROW INVOICE – 2019-212899 RW - ATD Signals Corpus Christi Dr 6700Blk modifications

**For the Permit emails sent from Folder Tab -> Options -> Print/Email, the subject line of the email should be set to the following format:**
-- ROW PERMIT – PERMIT NUMBER + TYPE – FOLDER NAME
-- For example, ROW PERMIT – 2019-212899 RW - ATD Signals Corpus Christi Dr 6700Blk modifications

**Comment by vschandramouli** [comment]
CC @Nadin-Nader

**Comment by TracyLinder** [comment]
@vschandramouli the AC is approved. Please move forward.

**Comment by vschandramouli** [comment]
AC reviewed by stakeholders. Moving to Backlog.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0316925 is submitted for CTM DEV deployment. 
Soon, I will submit another request for manual deployment of Permit reports email subject.............In progress.

**Comment by Nadin-Nader** [comment]
Waiting on SCTASK0316987 to be deployed to TEST

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I tested the DEV deployed for WAA process attemps - email subject format. 
For EX, RW, PSM it looks as expected, but for DS its still giving the old format. I can fix that when I am back.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0322048 is submitted TEST deployment request for SCTASK0316925.
[20d936ec7b49](http://coaamandadevops02.austintexas.gov/R5:20d936ec7b49e3c19edea98e5f90059c3ece8c2e) is submitted for validlookup (90245 - 197, 254, 253). Once approved, I will submit DEV/TEST deployment. Thanks.

**Comment by Nadin-Nader** [comment]
@SubhaShroff this has been deployed to DEV/TEST. Please go ahead and UAT this and i will put it on the 2pm agenda

**Comment by TracyLinder** [comment]
I asked Deanna to add the invoice update to the next AOB meeting agenda.

**Comment by TracyLinder** [comment]
### Approved to move to PROD 🎉
_without the invoice update / deferred to AOB._

**Comment by SubhaShroff** [comment]
@Nadin-Nader  
1. SCTASK0324667 - PROD Manual deployment request is raised.
2. SCTASK0324679 - PROD DevOps deployment request is raised.
Thanks.

**Comment by Nadin-Nader** [comment]
Message sent to Kyle asking him to UAT this on PROD

**Comment by TracyLinder** [comment]
### Update
- Kyle reporting that this feature isn't working correctly on EX.
- The problem is the subject line is **empty**

FYI @SubhaShroff and @Nadin-Nader this needs to be looked at.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  I see the PROD code for EX WAA is same as in TEST.
And I tested this today for EX on TEST. It is working. It could be permit specific. So please let me know the RSN so that I can debug this further.
Thanks.

**Comment by SubhaShroff** [comment]
I had a chat with Kyle and I realize its not about WAA process. With the screenshot/folderrsn provided by Kyle on Teams, I realize its the email subject thats going empty with Folder Tab -> Options -> Print/Email report.
Looks like the configuration I did was reverted to correct a bug that was adding ROW STATUS in subject instead of INVESTIGATION REPORT for one of the deficiency reports. I am going to check with Sree about it as she was going to work on that bug.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  SCTASK0330049 is submitted to set the EX permit report email subject on PROD.

**Comment by SubhaShroff** [comment]
@Nadin-Nader I verified the fix for email subject on PROD. If possible, I need your help to verify this before giving it to Kyle.

**Comment by Nadin-Nader** [comment]
@SubhaShroff  Kyle has approved this.
Closing
